.travis.yml

```diff
@@ -1,18 +1,22 @@
 language: php
 
+sudo: false
+
+cache:
+    directories:
+      - $HOME/.composer/cache
+
 before_script:
     # symfony/*
     - sh -c "if [ '$SYMFONY_DEPS_VERSION' = '2.8' ]; then sed -i 's/~2.6/2.8.*@dev/g' composer.json; composer update --dev --prefer-source; fi"
     - sh -c "if [ '$SYMFONY_DEPS_VERSION' = '2.7' ]; then sed -i 's/~2.6/2.7.*@dev/g' composer.json; composer update --dev --prefer-source; fi"
     - composer install --dev --prefer-source
 
-script: phpunit
-
 php:
-  - 5.5
-  - 5.6
-  - 7.0
-  - hhvm
+    - 5.5
+    - 5.6
+    - 7.0
+    - hhvm
```

Comment:
Can you remove this `php` section and move those version of PHP in the matrix? That simplifies the file.